python_deltabot_plugin/echo.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+from deltabot.hookspec import deltabot_hookimpl
+
+version = '0.5'
```

Comment:
since this is not a package that will be installed using pip and uploaded to PyPi, this version could be removed, it is not used. (not saying it is wrong to have it but users might think it is mandatory)